### DISCUSSION TITLE
Fall grammar engineering meetings

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#141] [Announcements]
Dear @delph-in/participants ,

If you are interested in our biweekly grammar engineering meeting, please fill out this when2meet. Please **ignore the specific dates** and **treat the suggested times as referring to generally odd and even weeks** in the Fall quarter/semester.

We will then choose **either** odd or even week day.

https://www.when2meet.com/?21323220-PgMyl

Please don't delay filling the poll out too much so I don't have to pester you later :).

**Comment by olzama** [comment]
@delph-in/participants please don't forget to fill out the poll, if you are interested :)

**Comment by olzama** [comment]
Dear @delph-in/participants :

We will try the following this Fall:

(1) On the first week of the month, a meeting on Thursday 5 PM Madrid time 
(2) On the third week of the month, a meeting on Monday 6 PM Madrid time

It's not ideal but I really didn't want to exclude any of the interested people for the entire semester. This way, some people could participate every time and some could join once a month.

I will post Zoom details here a few days before each meeting, and will send Google Calendar meetings to people. The next one will be on October 5.

**Comment by olzama** [comment]
Dear @delph-in/participants , a reminder about the next grammar engineering meeting: Monday October 16 18:00 Madrid time.

Zoom Meeting:

https://www.google.com/url?q=https://zoom.us/j/4768685765?pwd%3DNnZqQmx5QjcyNm94b1BuRUR3NUVkZz09&sa=D&source=calendar&usd=2&usg=AOvVaw2taiKUt_prWz7-ANV-Z9Fd

Meeting ID: 476 868 5765
Passcode: 459442

(This Fall, we alternate Mondays and Thursdays, to try and accommodate more people).

**Comment by danflick** [comment]
Hi Olga,

I am delayed today by another meeting that should end in about 10 minutes,
so if you're still there and want to talk, I'll be glad to join.  If not,
we can do it two weeks from now instead.  Sorry not to be on time!

 Dan

**Comment by fcbond** [comment]
several of us are here, please join.


On Mon, 16 Oct 2023 at 18:06, Dan Flickinger ***@***.***>
wrote:

> Hi Olga,
>
> I am delayed today by another meeting that should end in about 10 minutes,
> so if you're still there and want to talk, I'll be glad to join. If not,
> we can do it two weeks from now instead. Sorry not to be on time!
>
> Dan
>
> On Fri, Oct 13, 2023 at 3:44 AM Olga Zamaraeva ***@***.***>
> wrote:
>
> > Dear @delph-in/participants
> > <https://github.com/orgs/delph-in/teams/participants> , a reminder
> about
> > the next grammar engineering meeting: Monday October 16 18:00 Madrid
> time.
> >
> > Zoom Meeting:
> >
> >
> >
> https://www.google.com/url?q=https://zoom.us/j/4768685765?pwd%3DNnZqQmx5QjcyNm94b1BuRUR3NUVkZz09&sa=D&source=calendar&usd=2&usg=AOvVaw2taiKUt_prWz7-ANV-Z9Fd
> >
> > Meeting ID: 476 868 5765
> > Passcode: 459442
> >
> > (This Fall, we alternate Mondays and Thursdays, to try and accommodate
> > more people).
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/orgs/delph-in/discussions/141#discussioncomment-7271666>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AG5PC4QB4RRI42IPZ4FS6MTX7D5NLAVCNFSM6AAAAAA4U33O4SVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TENZRGY3DM>
>
> > .
> > You are receiving this because you are on a team that was
> mentioned.Message
> > ID: ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/discussions/141#discussioncomment-7294766>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRWIRR2ITSRG2USJMGDX7VLSFAVCNFSM6AAAAAA4U33O4SVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TEOJUG43DM>
> .
> You are receiving this because you are on a team that was mentioned.Message
> ID: ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by arademaker** [comment]
Sorry for missing today... what did I miss? BTW, we could try to make some notes about the meetings.

**Comment by olzama** [comment]
Dear all, a reminder that tomorrow there is a grammar engineering meeting at 17:00 Madrid time: 

https://zoom.us/j/4768685765?pwd=NnZqQmx5QjcyNm94b1BuRUR3NUVkZz09

Meeting ID: 476 868 5765
Passcode: 459442

**Comment by danflick** [comment]
Sorry I won't be able to join today.

Dan

**Comment by olzama** [comment]
@delph-in/participants Reminder about the grammar engineering meeting, for those who are interested:

Thursday Nov 30 17:00 Madrid time

[https://zoom.us/j/4768685765?pwd=NnZqQmx5QjcyNm94b1BuRUR3NUVkZz09](https://www.google.com/url?q=https://zoom.us/j/4768685765?pwd%3DNnZqQmx5QjcyNm94b1BuRUR3NUVkZz09&sa=D&source=calendar&usd=2&usg=AOvVaw2taiKUt_prWz7-ANV-Z9Fd)

Meeting ID: 476 868 5765
Passcode: 459442

**Comment by olzama** [comment]
Last Fall GE meeting today, Monday December 11, 18:00 Madrid time! For all interested @delph-in/participants 

After that, I will send out a poll regarding the Winter meetings :).

[https://zoom.us/j/4768685765?pwd=NnZqQmx5QjcyNm94b1BuRUR3NUVkZz09](https://www.google.com/url?q=https://zoom.us/j/4768685765?pwd%3DNnZqQmx5QjcyNm94b1BuRUR3NUVkZz09&sa=D&source=calendar&usd=2&usg=AOvVaw2taiKUt_prWz7-ANV-Z9Fd)

Meeting ID: 476 868 5765
Passcode: 459442
